plugin/session_buffer.py

```diff
@@ -308,15 +309,15 @@ def on_revert_async(self, view: sublime.View) -> None:
 
     on_reload_async = on_revert_async
 
-    def purge_changes_async(self, view: sublime.View) -> None:
+    def purge_changes_async(self, view: sublime.View, suppress_requests: bool = False) -> None:
```

Comment:
can we remove `suppress_requests`? 
As it is not used currently (it is always `False`).

Reply:
Sorry, yes I already noticed that I missed to use this argument in the function call, but I forgot to push the corrected version. The intention of this argument is that we don't need to send all the requests to the server when we know that the view is closing.